src/firebase/utils/getData.js

```diff
@@ -0,0 +1,12 @@
+import db from '../firebase';
+
+const getData = async (collectionName, data) => {
+  db.collection(collectionName).get(data)
+    .then(() => console.log('Document successfully added!'))
+    .catch((error) => {
+      console.error('Error writing document: ', error);
+      return false;
```

Comment:
Why is there a return false, theoretically this is an async function so I don't think this return goes back anywhere.